components/Cards/ResourceCard/ResourceSkeletonCard.tsx

```diff
@@ -1,15 +1,17 @@
-import { number } from 'prop-types';
 import { SKELETON_CARD } from '../../../common/constants/testIDs';
 import styles from './ResourceSkeletonCard.module.css';
 
-ResourceSkeletonCard.propTypes = {
-  numberOfSkeletons: number.isRequired,
+export type ResourceSkeletonCardPropsType = {
+  /**
+   * Sets the number of UI "skeletons" to render.
+   */
+  numberOfSkeletons: number;
 };
 
-function ResourceSkeletonCard({ numberOfSkeletons }) {
+function ResourceSkeletonCard({ numberOfSkeletons }: ResourceSkeletonCardPropsType) {
   return (
     <div className={styles.resourcesCardWrapper}>
-      {[...Array(numberOfSkeletons)].map((skeleton, index) => (
+      {[...Array(numberOfSkeletons)].map((index) => (
```

Comment:
whoops this is a mistake. we want to keep it as it was. the first argument is always undefined.